.editorconfig

```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
I don't refuse to add those editor configurations, but this is unrelated to type updates. The source code is formatted by the prettier, so this is only for the editor's benefit (it does not add protection to the source code).

Please move those files to a separate PR.

Reply:
@Qsppl I meant all of the editor files, `vscode` extensions too.

Reply:
Yes, I missed it. I also removed `.gitattributes`.